README.md

```diff
@@ -70,7 +70,46 @@ The following camera and microphone permissions need to be granted to enable aud
 
 More details on how to request permissions can be found in [Android](https://developer.android.com/training/permissions/requesting) and [iOS](https://developer.apple.com/documentation/avfoundation/cameras_and_media_capture/requesting_authorization_for_media_capture_on_ios?language=objc) official documents.
 
-## 3. Connect React Native code to the Amazon Chime SDK
+## 3. Add libraries for SDK
+
+### Android
+You'll need following libraries to be able to build the Android application.
+From `build.gradle`
+```
+buildscript {
+    ext {
+        kotlin_version = '1.3.71'
+        minSdkVersion = 21
+        compileSdkVersion = 29
+        targetSdkVersion = 29
+    }
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+    }
+allprojects {
+    repositories {
+        flatDir {
+            dirs 'libs'
+        }
+    }
+}
+```
+
+From `app/build.gradle`
+```
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+dependencies {
+    implementation(name: 'amazon-chime-sdk', ext: 'aar')
+    implementation(name: 'amazon-chime-sdk-media', ext: 'aar')
+    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.3'
+    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.3'
+    implementation 'com.google.code.gson:gson:2.8.6'
```

Comment:
Why do we require these three?

Reply:
Our framework doesn't seem to contain these libraries by default, so need to add it.